ipywidgets/static/widgets/js/widget_string.js

```diff
@@ -128,11 +128,7 @@ define([
             /**
              * Set a css attr of the widget view.
              */
-            if (name == 'padding' || name == 'margin') {
-                this.$el.css(name, value);
-            } else {
-                this.$textbox.css(name, value);
-            }
+            this.$el.css(name, value);
```

Comment:
I think the base class defines this method, exactly like this.  No need to redefine it here - go ahead and delete the whole method.